VirtoCommerce.Platform.Core/Extensions/DIExtension.cs

```diff
@@ -0,0 +1,23 @@
+using System.Linq;
```

Comment:
Why do you need this extension?  Could you explain?

Reply:
There are two implementation: PlatformEmailSender(with NotImplementException) and NotificationEmailSender(with true implementation) in the DI container. If we install the NotificationsModule then we use  NotificationEmailSender instead of PlatformEmailSender. So, we replace the implementation. This one of kind replacing of an implementation.

Reply:
It is working, I cheked